test/services/entity_builder.js

```diff
@@ -430,6 +434,18 @@ describe('Entity Builder', () => {
       expect(() => entityBuilder.validateAndCastFindAssetsParams({perPage: '1.3'})).to.throw(ValidationError);
       expect(() => entityBuilder.validateAndCastFindAssetsParams({perPage: '-10'})).to.throw(ValidationError);
     });
+
+    it('throws if fromTimestamp is not positive integer', async () => {
+      expect(() => entityBuilder.validateAndCastFindAssetsParams({fromTimestamp: 'avc'})).to.throw(ValidationError);
+      expect(() => entityBuilder.validateAndCastFindAssetsParams({fromTimestamp: '1.3'})).to.throw(ValidationError);
+      expect(() => entityBuilder.validateAndCastFindAssetsParams({fromTimestamp: '-10'})).to.throw(ValidationError);
+    });
+
+    it('throws if toTimestamp is not positive integer', async () => {
+      expect(() => entityBuilder.validateAndCastFindAssetsParams({toTimestamp: 'avc'})).to.throw(ValidationError);
+      expect(() => entityBuilder.validateAndCastFindAssetsParams({toTimestamp: '1.3'})).to.throw(ValidationError);
+      expect(() => entityBuilder.validateAndCastFindAssetsParams({toTimestamp: '-10'})).to.throw(ValidationError);
```

Comment:
Cases we check: string, non-integer value, negative value. Maybe also add `{}` and `[]`?

Reply:
I think it should be tested in validators test. Testing any possible value for any possible parameter is not possible anyway